### PR TITLE
Task retries and ingress timeout

### PR DESCRIPTION
**Pull request by sebader**
Learning from today's incident:
- Ingress ingress timout to higher than default of 60seconds in order to see actual errors from the backend instead of 504 Gateway timeouts
- Retry most of our ADO tasks at least once. Doesnt hurt and might "solve" many transient issues

### CODE REVIEW

File: .ado/pipelines/templates/jobs-workload-deploy.yaml [hunk 1]
```diff
@@ -160,6 +162,7 @@ jobs: # using jobs so they each run in parallel
 
     - task: AzureCLI@2
       displayName: 'Install workload BackgroundProcessor on AKS clusters'
```
Comment (heoelri) [inline .ado/pipelines/templates/jobs-workload-deploy.yaml:164]: not sure if you want this set in all tasks for consistency - we've already inline retry logic for the deployment of catalogservice and backgroundprocessor (due to the issues with cert-manager)

Comment (sebader) [reply .ado/pipelines/templates/jobs-workload-deploy.yaml:164]: hm valid point. would would you prefer?

Comment (heoelri) [reply .ado/pipelines/templates/jobs-workload-deploy.yaml:164]: it probably makes sense to set it consistenly across all tasks as you've suggested - inline is still required when a delay between two retries is required

Comment (sebader) [reply .ado/pipelines/templates/jobs-workload-deploy.yaml:164]: ok lets keep it as is then

**Review by heoelri — APPROVED** [review]

**Pull request merged**